pages/src/utils/childDocumentsOf.ts

```diff
@@ -0,0 +1,77 @@
+import { PayloadRequest, CollectionSlug } from 'payload'
+import type { SanitizedCollectionConfig } from 'payload'
+import type { PageCollectionConfigAttributes } from '../types/PageCollectionConfigAttributes.js'
+import type { PagesPluginConfig } from '../types/PagesPluginConfig.js'
+
+async function childDocumentsOfInternal(
```

Comment:
Could you please explain why you added a `childDocumentsOf` and `childDocumentsOfInternal`? 

Please add a doc comment to this function and ensure it's exported in the `index.ts` file.